wrangler.jsonc

```diff
@@ -22,6 +22,12 @@
       },
     ],
   },
+  "r2_buckets": [
+    {
+      "binding": "PDF_BUCKET",
+      "bucket_name": "loresmith-pdfs",
```

Comment:
we shouldn't do it this way if we want to be able to pick what campaigns we are sourcing from; i think we should ask users to name their campaign and have a bucket per campaign. also, i think the should be user based too (`loresmith/ofisk/obojima` type structure)

Reply:
Also, they definitely should name their campaign, imo. Would be helpful with tons of stuff that will be tightly coupled to campaign.